### ISSUE TITLE
Next Waitlist app not owned by GitHub

### ISSUE THREAD

**Issue by lennax** [githubnext/githubnext#5]
I clicked the waitlist button for https://githubnext.com/projects/copilot-cli/ and was presented with an app authorization screen stating that the Next Waitlist app is not owned or operated by GitHub:

<img width="578" alt="next_waitlist" src="https://user-images.githubusercontent.com/952658/226962317-9940b0ae-ad32-472a-a319-aa053bca5233.png">

Expected result:

The Next Waitlist app should show that it's owned and operated by GitHub.

**Comment by v-antech** [comment]
Yes, that's actually what stopped me from activating. This "Act on your behalf" is something I agree to only very begrudgingly.

**Comment by lukesterb183** [comment]
What in the wide world of sports is happening here and how did my dumbass get in this coding stuff.
